### DISCUSSION TITLE
How to launch Fluent with Pyfluent using Remote Linux Nodes

### DISCUSSION THREAD

**Discussion by rkotopbas** [ansys/pyfluent#2644] [Q&A]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

I do my analysis from my personal computer with Pyfluent. But I want to use fluent's remote feature and start it on another computer with 'Use Remote Linux Nodes'. I don't know how to enter the 'Remote Fluent Root Path', 'Remote Working Directory' and 'Use Remote Cluster Head Note' sections that I entered in the Remote tab. That's why I can't work remotely. Can you help me?

### 📝 Steps to reproduce

pyfluent.launch_fluent(precision="double", processor_count ="processor_count", mode="meshing")

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2023 R1

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
-
```

**Comment by rkotopbas** [comment]
up up

**Comment by ehsansadeghi1** [comment]
i have the same issue. I am using ansys 2024R2
